packages/calcite-components/src/components/rating/rating.e2e.ts

```diff
@@ -766,4 +770,65 @@ describe("calcite-rating", () => {
       expect(element).toEqualAttribute("value", "2");
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-rating-spacing": {
+          targetProp: "--calcite-internal-rating-spacing",
+        },
+        "--calcite-rating-spacing-unit": {
+          targetProp: "--calcite-internal-rating-spacing",
+        },
+        "--calcite-internal-rating-spacing": {
+          shadowSelector: `.${CSS.fieldset}`,
+          targetProp: "gap",
+        },
+        "--calcite-rating-outline-color": {
+          shadowSelector: `.${CSS.star}`,
+          targetProp: "color",
+        },
+        "--calcite-rating-outline-color-hover": {
+          shadowSelector: `.${CSS.star}`,
+          targetProp: "color",
+          state: "hover",
+        },
+        "--calcite-rating-value-fill-color": {
+          shadowSelector: `.${CSS.star}`,
+          targetProp: "color",
+          state: { press: { attribute: "class", value: "star" } },
+        },
+      };
+      themed(html`<calcite-rating></calcite-rating>`, tokens);
```

Comment:
Forgot to mention this before, but if you wanted to simplify, common test helpers also support a tag to set up a basic component page:

```ts
themed("calcite-rating", tokens);
```

Up to you.